lib/dataSource.js

```diff
@@ -16,138 +16,123 @@ var fluid = require("infusion"),
     kettle = fluid.registerNamespace("kettle"),
     fs = require("fs"),
     http = require("http"),
-    url = require("url");
+    urlModule = require("url"),
```

Comment:
Pending full resolution to FLUID-5900, I've at least split the file into browser-style dataSource-core.js and the rest into dataSource-node.js